test-data/unit/check-default-plugin.test

```diff
@@ -0,0 +1,33 @@
+-- Test cases for the default plugin
+--
+-- Note that we have additional test cases in pythoneval.test (that use real typeshed stubs).
```

Comment:
Maybe, though there is some value in having a few test cases in `pythoneval.test`, since they use typeshed stubs which may work differently from the test fixtures. My take is that `pythoneval.test` should only have at most a few, coarse-grained test cases per feature, whereas in `check-*.test` we can have large number of test cases that test various edge cases etc., since these tests are much faster.